src/Datasource/Log/ElasticLogger.php

```diff
@@ -118,6 +118,9 @@ public function log(mixed $level, Stringable|string $message, array $context = [
     protected function _log(string $level, string $message, array $context = []): void
     {
         $logData = $context;
+        if ($level === LogLevel::INFO || $message !== 'Elastica Request') {
```

Comment:
What other kinds of logs are we dropping here?

Reply:
It coming from transport layer, so we getting 5 log entries per one request https://github.com/elastic/elastic-transport-php/blob/8be37d679637545e50b1cea9f8ee903888783021/src/Transport.php#L277

```
ROOT\src\Datasource\Log\ElasticLogger.php (line 121)
########## DEBUG ##########
'Request: GET http://localhost:9200/_stats'
###########################
ROOT\src\Datasource\Log\ElasticLogger.php (line 121)
########## DEBUG ##########
'Headers: {"Host":["localhost:9200"]}
Body: '
###########################
ROOT\src\Datasource\Log\ElasticLogger.php (line 121)
########## DEBUG ##########
'Response (retry 0): 200'
###########################
ROOT\src\Datasource\Log\ElasticLogger.php (line 121)
########## DEBUG ##########
'Headers: {"X-elastic-product":["Elasticsearch"],"content-type":["application\/json"],"Transfer-Encoding":["chunked"]}
Body: {"_shards":{"total":8,"successful":4,"failed":0},"_all":{"primaries":{"docs":{"count":100,"deleted":0,"total_size_in_bytes":1048486},"shard_stats":{"total_count":4},"store":{"size_in_bytes":1049279,"total_data_set_size_in_bytes":1049279,"reserved_in_bytes":0},"indexing":{"index_total":0,...
```